pdcupdater/utils.py

```diff
@@ -633,7 +641,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def retry(timeout=500, interval=20, wait_on=Exception):
+def retry(timeout=300, interval=10, wait_on=Exception):
```

Comment:
Don't adjust the defaults, since I assume these are used elsewhere.  Just pass your preferred values to the *application* of the ``@retry(...)`` decorator.